public/styles.css

```diff
@@ -994,154 +994,496 @@ input[type=number]::-webkit-outer-spin-button {
 .trades {
     overflow: auto;
 }
-
+/*
 .trades, .layout-btn {
     flex-wrap: wrap;
+}*/
+.layout-btn {
+    flex-wrap: wrap;
 }
-
 .layout-mini {
     background-color: white;
     border-radius: 1px;
     margin: 1px;
 }
 
-.layout-1-1 {
+.layout-mini-1-1 {
```

Comment:
That is not how we want to do it.
The mini button and the big layout work in the exact way, and we should be able to use the same styles. This change adds 300-400 lines of styles!!!